espnet/asr/pytorch_backend/asr.py

```diff
@@ -202,7 +202,6 @@ def update_core(self):
         if self.grad_noise:
             from espnet.asr.asr_utils import add_gradient_noise
             add_gradient_noise(self.model, self.iteration, duration=100, eta=1.0, scale_factor=0.55)
-        loss.detach()  # Truncate the graph
```

Comment:
is it OK?

Reply:
@ShigekiKarita and I discussed about this before: https://github.com/espnet/espnet/pull/1421#discussion_r355423179

The removed line uses a non-inplace operation and does not make any difference.
It meant to detach the variable `loss`. But it is not used in the rest part of `update_core()`.

Reply:
ok